graph/db/models/channel_edge_policy.go

```diff
@@ -21,9 +20,6 @@ type ChannelEdgePolicy struct {
 	// use SetSigBytes instead to make sure that the cache is invalidated.
```

Comment:
nit: docs still mentions the cache, perhaps we should now set `SigBytes` directly and remove `SetSigBytes`?